spec/unit/block_score/invalid_request_error_spec.rb

```diff
@@ -1,10 +1,9 @@
-module BlockScore
-  RSpec.describe InvalidRequestError do
-    subject { -> { BlockScore::Person.create(first_name: 'John') } }
-    let(:message) do
-      '(Type: invalid_request_error) One or more required parameters are invalid (name_first) (Status: 400)'
-    end
-
-    it_behaves_like 'an error'
+RSpec.describe BlockScore::InvalidRequestError do
+  subject { -> { BlockScore::Person.create(first_name: 'John') } }
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/68/files#r52838287